docs/transformer.md

```diff
@@ -333,3 +333,30 @@ To generate a tutorial:
 1. In each repository, commit your changes, push your branch, and open a pull request.
 
 1. A Killercoda maintainer reviews the PR to ensure that the generate tutorial works as expected.
+
+
+## Scripts and Extra Course Files
+
+If your tutorial requires scripts or extra files, make sure to manually add them to the tutorial directory in the Killercoda repository. For example, if your tutorial requires a script to run:
+
+1. Add the bash script to the tutorial directory in the Killercoda repository. See the [what-is-loki](../loki/what-is-loki/) tutorial for an example.
+2. Add the script to the `scripts` field in the `index.json` file:
```

Comment:
You can use repetitive numbering and let the renderer auto increment for you which makes it easier to update steps: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists

I'm a bit confused by the example because it doesn't reference the `scripts` field. JSON is very hard to demonstrate in the docs though and you have to be very verbose to help a user: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#partial-code-examples so I understand there may not be much you can do

Reply:
This is really good point let me flesh this out a little better!